pkg/sfu/buffer/dependencydescriptorparser.go

```diff
@@ -66,12 +67,15 @@ type ExtDependencyDescriptor struct {
 	ActiveDecodeTargetsUpdated bool
 	Integrity                  bool
 	ExtFrameNum                uint64
+	// the frame number of the keyframe which the current frame depends on
```

Comment:
technically, current frame depends on the dependency descriptor structure attached to this key frame, right? not directly the key frame itself. Not a big issue, just trying to understand.

Reply:
Curiosity question: is a structure always attached to a key frame only? Wondering if this should be renamed to `ExtStructureFrameNum` to mean frame number to which this structure was attached.

Reply:
Good question, I checked the specification that it doesn't require a structure must be carried by keyframe, but in libwebrtc's frame assembler, it assume that every frame attach a structure is a key frame.